### DISCUSSION TITLE
Question est-ce que l'opensource va être donnée complétement ?

### DISCUSSION THREAD

**Discussion by Adrien070503** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#153] [General]
Bonjour, 

L'opensource avec le server ayant fermé définitivement va t-il être rendu publique en mode complet ?

**Comment by DreamXZE** [comment]
Le serveur qui ferme était une blague de premier avril, SoZ est toujours ouvert et actif 🙏

**Reply by Adrien070503** [reply]
Okay :) j'avais pas compris du tout XD
